testinfra/host.py

```diff
@@ -16,13 +16,13 @@
 import testinfra.modules
 
 
-class Host(object):
+class Host:
     _host_cache = {}
     _hosts_cache = {}
 
     def __init__(self, backend):
         self.backend = backend
-        super(Host, self).__init__()
+        super().__init__()
```

Comment:
I'm not sure this line is actually needed.

Reply:
Not sure too, but I think it's better to have this in case we add a parent class later.